.github/workflows/test.yaml

```diff
@@ -0,0 +1,42 @@
+name: Testing
```

Comment:
Will not it a bit confusing that the 'Testing' workflow does not run tests and just checks, whether the code compiles?

Reply:
renamed to 'building' (I still hope that some day we will add an integration testing of new jepsen changes vs tarantool)